packages/modules/order/tsconfig.json

```diff
@@ -8,5 +8,8 @@
       "@types": ["./src/types"],
       "@utils": ["./src/utils"]
     }
+  },
+  "ts-node": {
+    "transpileOnly": true
   }
```

Comment:
q: shouldn't this be on the base config?

Reply:
Needs to be removed altogether. Slipped from some custom testing